internal/pkg/defines.go

```diff
@@ -62,7 +69,8 @@ type Storable interface {
 	Insert(ctx context.Context, userID int, url, code string) error
 	InsertBatch(ctx context.Context, userID int, batch [][]string) error
 	DeleteBatch(ctx context.Context, userID int, batch []string)
-	Count() int
+	Count(ctx context.Context) int
+	GetUniqueUsers(ctx context.Context) []int
```

Comment:
```suggestion
	GetUniqueUsers(ctx context.Context) (int64, error)
```
Если записей будет хотя бы сотня тысяч, начнутся заметные тормоза. И не забываем про ошибки выполнения.